cromwell_tools/__init__.py

```diff
@@ -0,0 +1,6 @@
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    __version__ = get_distribution(__name__).version
```

Comment:
Don't we need to set a version variable somewhere in cromwell tools for this to work? We are reading `.version` here, but how does it get set?

Reply:
It is set internally. For all git based packages, this setuptools_scm lib will "magically" grab the git tag/branch/commit-hash for you, no matter where you install the package (through PyPI or git+git). 

So for example, if now I `pip install .` the cromwell_tools from my git branch, `cromwell_tools.__version__` will give me: something like `'0.2.1.dev4+gf1682bf.d20180418'`. After I merge in this PR and tag the Cromwell-tools with `0.3.0`, if Lira install it through `git+git://github.com/broadinstitute/cromwell-tools.git@v0.3.0`, the `__version__` will give `0.3.0`.